src/UnitTests/Core/Commands/TopBallerCommandTests/ProcessShould.cs

```diff
@@ -0,0 +1,126 @@
+using DevChatter.Bot.Core.Commands;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Data.Specifications;
+using DevChatter.Bot.Core.Events.Args;
+using DevChatter.Bot.Core.Systems.Chat;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace UnitTests.Core.Commands.TopBallerCommandTests
+{
+    public class ProcessShould
+    {
+        private TopBallersCommand _topBallersCommand;
+        private readonly Mock<IChatClient> _chatClientMock = new Mock<IChatClient>();
+        private readonly Mock<IRepository> _repositoryMock = new Mock<IRepository>();
+
+        private void SetUpTest(List<ChatUser> ballers)
+        {
+            _repositoryMock.Setup(x => x.List(It.IsAny<ISpecification<ChatUser>>())).Returns(ballers);
+            _topBallersCommand = new TopBallersCommand(_repositoryMock.Object);
+        }
+
+        [Fact]
+        public void ReturnFiveBallers_GivenTestData()
+        {
+            List<ChatUser> testBallers = GetTestUser();
+            SetUpTest(testBallers);
+            List<ChatUser> actualBallers = _topBallersCommand.TopFiveBallers();
+            Assert.Equal(5, actualBallers.Count);
+        }
+
+        [Fact]
+        public void ReturnChatUserwithHighestTokens_GivenTestData()
+        {
+            List<ChatUser> testBallers = GetTestUser();
+            SetUpTest(testBallers);
+            List<ChatUser> actualBallers = _topBallersCommand.TopFiveBallers();
+            Assert.Equal(testBallers.Max(b => b.Tokens), actualBallers[0].Tokens);
+        }
+
+        // Test Data has 6 users
+        // This test ensure the sixth user (lowest tokens) is not in the list
+        [Fact]
+        public void EnsureSixthHighestTokenBallerIsNotIncludedInList_GivenTestData()
+        {
+            List<ChatUser> testBallers = GetTestUser();
+            SetUpTest(testBallers);
+            List<ChatUser> actualBallers = _topBallersCommand.TopFiveBallers();
+            Assert.NotEqual(testBallers.Min(b => b.Tokens),  actualBallers.Min(b => b.Tokens));
+        }
+
+        [Fact]
+        public void EnsureMessageIsCorrect_GivenTestData()
+        {
+            List<ChatUser> testBallers = GetTestUser();
+            SetUpTest(testBallers);
+            List<ChatUser> actualBallers = _topBallersCommand.TopFiveBallers();
+            string testMessage = TestGenerateMessage(actualBallers);
+            string actualMessage = _topBallersCommand.GenerateMessage(actualBallers);
+            Assert.Equal(testMessage, actualMessage);
```

Comment:
I would do an assert contains for the right people's names rather than an exact string. This will be brittle by only checking a specific string.